face-kiosk/face-kiosk-application/src/main/java/com/pillar/skynet/facekiosk/application/Main.java

```diff
@@ -43,11 +50,21 @@ public static void main(String[] s) throws IOException, FrameGrabber.Exception {
 			LOG.info("Found a face tho.");
 		}
 
+		CvRect r = new CvRect(cvGetSeqElem(faces, 0));
+		cvSetImageROI(grayPhoto, r);
```

Comment:
ROI = Region of Interest. OpenCV will do things with only this particular region of the image.